plugins/in_node_exporter_metrics/ne.c

```diff
@@ -740,6 +763,26 @@ static int in_ne_init(struct flb_input_instance *in,
                     }
                     ne_processes_init(ctx);
                 }
+                else if (strncmp(entry->str, "thermal_zone", 12) == 0) {
+                    if (ctx->thermalzone_scrape_interval == 0) {
+                        flb_plg_debug(ctx->ins, "enabled metrics %s", entry->str);
+                        metric_idx = 14;
```

Comment:
Could you increment the index value to 15? This is because my patch's metrics is injected as index of 14.